File: fixtures/node_output/post_escaped_double_quotes_in_single_quotes.js
var request = require('request');

var dataString = 'foo=\"bar\"';

var options = {
    url: 'http://example.com/',
    method: 'POST',
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: fixtures/elixir/post_escaped_double_quotes_in_single_quotes.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://example.com/",
  options: [],
  headers: [],
  params: [],
  body: [
    {~s|foo|, ~s|\\"bar\\"|}
  ]
}

response = HTTPoison.request(request)
